### PR TITLE
@mzikherman: Redis => Memcached

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/artsy/metaphysics/issues/66

### CODE REVIEW

**Comment by dzucconi** [comment]
We'll use Memcachier on Heroku for now; which is about half the price of our Redis instance

**Comment by mzikherman** [comment]
ace :+1:

File: lib/cache.js [hunk 1]
```diff
@@ -29,15 +40,17 @@ export default {
...
     const timestamp = new Date().getTime();
-    if (_.isArray(data)) {
+    if (isArray(data)) {
       data.map(datum => datum.cached = timestamp); // eslint-disable-line no-param-reassign
     } else {
       data.cached = timestamp; // eslint-disable-line no-param-reassign
     }
 
-    return client.set(key, JSON.stringify(data));
+    return client.set(key, JSON.stringify(data), err => {
```
Comment (mzikherman) [inline lib/cache.js]: ah there's that 'required' callback

Comment (dzucconi) [inline lib/cache.js]: Actually I hadda switch libs cuz of Memcachier's reliance on the binary protocol; this one is less annoying. But just passing along the error for sanity's sake

**Pull request merged**